tests_integration/__tests__/early-exit.js

```diff
@@ -30,3 +30,11 @@ test("throw error with --find-config-path + multiple files", () => {
   expect(result.stderr).toMatchSnapshot();
   expect(result.status).toEqual(1);
 });
+
+test("throw error and show usage with something unexpected", () => {
+  const result = runPrettier("cli", [], { isTTY: true });
+
+  expect(result.stdout).toMatchSnapshot();
+  expect(result.stderr).toMatchSnapshot();
```

Comment:
Could we pass an argument to `toMatchSnapshot` to make it clearer in the diffs what we're looking at?

> `throw error and show usage with something unexpected 2` 

is not particularly useful.

Reply:
Passing the snapshot name directly is not a good idea since Jest will treat it as `<snapshot-name>` instead of `<test-title> <snapshot-name>`, maybe we should change them into something like this?

```js
describe("throw error and show usage with something unexpected", () => {
  const result = runPrettier("cli", [], { isTTY: true });
  test("stdout", () => {
    expect(result.stdout).toMatchSnapshot();
  });
  test("stderr", () => {
    expect(result.stderr).toMatchSnapshot();
  });
  test("status", () => {
    expect(result.status).not.toEqual(0);
  });
});
```